### PR TITLE
Renames Northern Star to Virgo

### PR DESCRIPTION
**Pull request by Arokha**
As it says on the tin.

### CODE REVIEW

**Comment by SpadesNeil** [comment]
The star is Erigone. :P

The star system is the Virgo-Erigone system. Virgo constellation, Erigone is the star.

File: code/game/machinery/exonet_node.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/exonet_node
 	name = "exonet node"
-	desc = "This machine is one of many, many nodes inside Vir's section of the Exonet, connecting the Northern Star to the rest of the system, at least \
+	desc = "This machine is one of many, many nodes inside Vir's section of the Exonet, connecting Virgo Station to the rest of the system, at least \
```
Comment (SpadesNeil) [inline code/game/machinery/exonet_node.dm]: This is incorrect for a number of reasons.

Vir is the name of the Northern Star. That should be changed to Erigone.

Furthermore, "Virgo Station" should be replaced with [station_name] or NSS Adephagia (our official moniker that no one but CentCom uses).

File: code/game/machinery/camera/camera_assembly.dm [hunk 2]
```diff
@@ -80,7 +80,7 @@
 			if(isscrewdriver(W))
 				playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
 
-				var/input = sanitize(input(usr, "Which networks would you like to connect this camera to? Separate networks with a comma. No Spaces!\nFor example: Northern Star,Security,Secret ", "Set Network", camera_network ? camera_network : NETWORK_EXODUS))
+				var/input = sanitize(input(usr, "Which networks would you like to connect this camera to? Separate networks with a comma. No Spaces!\nFor example: Virgo Station,Security,Secret ", "Set Network", camera_network ? camera_network : NETWORK_EXODUS))
```
Comment (SpadesNeil) [inline code/game/machinery/camera/camera_assembly.dm]: This is fine as it is.

File: code/global.dm [hunk 3]
```diff
@@ -25,14 +25,14 @@ var/global/list/global_map = null
 var/list/hit_appends = list("-OOF", "-ACK", "-UGH", "-HRNK", "-HURGH", "-GLORF")
 var/diary				= null
 var/href_logfile        = null
-var/station_name        = "Northern Star"
-var/station_short       = "Northern Star"
-var/const/dock_name     = "Vir Interstellar Spaceport"
+var/station_name        = "Virgo Orbital Research Establishment"
+var/station_short       = "VORE"
+var/const/dock_name     = "Virgo Central Command"
```
Comment (SpadesNeil) [inline code/global.dm]: These are all correct.

File: code/modules/clothing/spacesuits/spacesuits.dm [hunk 4]
```diff
@@ -48,7 +48,7 @@
 
 /obj/item/clothing/suit/space
 	name = "Space suit"
-	desc = "A suit that protects against low pressure environments. \"NORTHERN STAR\" is written in large block letters on the back."
+	desc = "A suit that protects against low pressure environments. \"NSS VORE\" is written in large block letters on the back."
```
Comment (SpadesNeil) [inline code/modules/clothing/spacesuits/spacesuits.dm]: NSS Adephagia

File: code/modules/examine/descriptions/telecomms.dm [hunk 5]
```diff
@@ -30,7 +30,7 @@
 	Exonets at the root node(s), and is typically arranged in a tree structure.  The root node(s) are generally government-owned and are very secure \
 	and resilient to failure.<br>\
 	<br>\
-	This node is privately owned and maintained by Nanotrasen, and allows the colonists of the Northern Star to have access to the Exonet."
+	This node is privately owned and maintained by Nanotrasen, and allows the users on Virgo Station to have access to the Exonet."
```
Comment (SpadesNeil) [inline code/modules/examine/descriptions/telecomms.dm]: Should say "allows inhabitants of the star system to have access to the exonet." The telecomms services more than just our station. It affects the planet Virgo-Prime, Central Command, _and_ the station we work on.

**Comment by Arokha** [comment]
Good to know, though I don't think that's referenced. Their station was literally called Northern Star. Their codebase is Polaris but as far as I know, the word isn't actually used in their game.

File: code/modules/clothing/spacesuits/spacesuits.dm (hunk 4 above)
Comment (Arokha) [inline code/modules/clothing/spacesuits/spacesuits.dm]: Our current one says NSS VORE, so this is different. Just checking to make sure that's what you want.

**Comment by Arokha** [comment]
Genericized this instead and PR'd it to Polaris. Will cherrypick it to here.

**Pull request closed without merging**